onedrive_service/tox.ini

```diff
@@ -34,7 +36,9 @@ commands = {posargs:flake8 '{toxinidir}/setup.py' '{toxinidir}/src' '{toxinidir}
 
 [testenv:pylint]
 deps =
+    mock
     pylint>=1.7,<1.8
+    -r{toxinidir}/requirements.txt
```

Comment:
Please, remove it, Tox installs the package automatically and `requirements.txt` are managed by `setup.py`.